<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pyramid_Tic_Tac_Toe</class>
 <widget class="QMainWindow" name="Pyramid_Tic_Tac_Toe">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pyramid Tic Tac Toe</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="noOfMovesLabel">
          <property name="maximumSize">
           <size>
            <width>677</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>outrun future</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="text">
           <string>NUMBER OF MOVES =  </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="P_TTT_Grid">
          <property name="minimumSize">
           <size>
            <width>677</width>
            <height>407</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>677</width>
            <height>407</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">P_TTT_Grid::item: {
  border: 2px solid red;
  border-radius: 5px;
}

hover: {
	background-color: transparent;
}</string>
          </property>
          <property name="autoScroll">
           <bool>false</bool>
          </property>
          <property name="tabKeyNavigation">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="rowCount">
           <number>3</number>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>135</number>
          </attribute>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string/>
           </property>
           <property name="background">
            <brush brushstyle="NoBrush">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="player1Label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Player 1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="windowModality">
             <enum>Qt::WindowModality::ApplicationModal</enum>
            </property>
            <property name="font">
             <font>
              <family>outrun future</family>
              <italic>false</italic>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #B0BEC5;</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>4</number>
            </property>
            <property name="midLineWidth">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name1Label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mark1Label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Mark: </string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="state1label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>State:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="styleSheet">
           <string notr="true">color: #B0BEC5;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="styleSheet">
           <string notr="true">color: #B0BEC5;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="player2Label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Player 2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="styleSheet">
             <string notr="true">color: #B0BEC5;</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>4</number>
            </property>
            <property name="midLineWidth">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="name2label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="mark2Label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Mark:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="state2Label">
            <property name="font">
             <font>
              <family>outrun future</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>State:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
